### PR TITLE
Release/13.7.0

### PR DESCRIPTION
**Pull request by annakubackaAnku**

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [13.7.0] - 2023-07-06
+
+### Added
+
+- **mi-floor-selector** is a new component for showing the "floor-selector".
```
Comment (ammapspeople) [inline packages/components/CHANGELOG.md:12]:
```suggestion
- **mi-floor-selector** is a new component for showing a MapsIndoors  Floor Selector.
```

File: packages/components/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+- **mi-my-position** is a new component for showing the "my-position" button.
```
Comment (ammapspeople) [inline packages/components/CHANGELOG.md:13]:
```suggestion
- **mi-my-position** is a new component for showing a MapsIndoors My Position button.
```

File: packages/components/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+
+## [13.6.0] - 2023-07-06
+
+### Added
+
+- **mi-list-item-location** implemets `showExternalId` property.
```
Comment (ammapspeople) [inline packages/components/CHANGELOG.md:19]:
```suggestion
- **mi-list-item-location** implements `showExternalId` property.
```

File: packages/components/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [13.7.0] - 2023-07-06
+
+### Added
+
+- **mi-floor-selector** is a new component for showing a MapsIndoors  Floor Selector.
```
Comment (ammapspeople) [inline packages/components/CHANGELOG.md:12]:
```suggestion
- **mi-floor-selector** is a new component for showing a MapsIndoors Floor Selector.
```

😅

**Comment by annakubackaAnku** [comment]
@ammapspeople I added the typo changes and also created readme file for the my-position component, I forgot about that one 🙈

**Comment by annakubackaAnku** [comment]
@ammapspeople ready again ✅

**Review by ammapspeople — APPROVED** [review]
✅

**Pull request merged**